### ISSUE TITLE
[Request] Vials work with Drains

### ISSUE THREAD

**Issue by scottosx** [MightyPirates/Vials#9]
This is a great idea. It makes alloying and part assembly more precise. Overall nicely done.

I expected that it would work with the drains to pull molten metals directly. This is desirable to empty leftovers in the smeltery or pull needed amounts from large Tinkers Tanks (the multiblock structures)

Can this be added?

**Comment by lordjoda** [comment]
I'll have a look at it when I have time. I can't remember if I tried to implement it but it didn't work or I simply forgot it.

**Comment by scottosx** [comment]
@lordjoda Such a nice feature addition! It works very well, thank you. The only thing I notice now is that the vials won't pull liquids in Creative mode. They make the fluid 'gurgle' sound but nothing goes into your inventory.

OT, but since this is a closed issue and I have watched some of your OpenComputers tutorials...
I wonder what you think of this tutorial video I have created for the OC nanomachines? https://youtu.be/ljUKQ0o-qms  

Since most of it was based on my research I just want to be sure that the info is accurate. Sorry if this is the wrong place to ask.
